### PR TITLE
Fix #18

### LINKED ISSUE #18: README.md в корне репозитория
Написать краткую инструкцию по использованию программы => антизатуп

### PR DESCRIPTION
**Pull request by seele402**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,16 @@ spotify current track to csgo chat very nice
...
 
+# How to use
+1. Login to your Spotify account (Don't forget to use "Remember me" option)
+2. Select your Steam account in "current acc" list
+3. Enter desired chat button (If you run into any trouble setting one, you can press the "затуп?" button to see the Source engine keys)
+4. Optionally, you can use autosend, which activates send button on each change of track via WinAPI SendInput
+5. Turn on the programm by switching the "npc work" slider on
+6. Copy the console string
+7. Paste console string into the csgo console (~)
+8. Press the binded key to send current track info or use the autosend option metioned in the 4th step
```
Comment (moetelo) [inline README.md:14]: > binded

bound

Comment (seele402) [reply README.md:14]: assigned

File: README.md [hunk 2]
```diff
@@ -3,4 +3,16 @@ spotify current track to csgo chat very nice
 
 cefsharp & ie11 versions will work for 1 hour because refresh token logic isnt implemented yet
 
+# How to use
+1. Login to your Spotify account (Don't forget to use "Remember me" option)
+2. Select your Steam account in "current acc" list
+3. Enter desired chat button (If you run into any trouble setting one, you can press the "затуп?" button to see the Source engine keys)
+4. Optionally, you can use autosend, which activates send button on each change of track via WinAPI SendInput
+5. Turn on the programm by switching the "npc work" slider on
```
Comment (moetelo) [inline README.md:11]: > programm

program

Comment (seele402) [reply README.md]: иди нахуй

**Pull request merged**